cmake/Modules/CocosBuildHelpers.cmake

```diff
@@ -172,27 +178,19 @@ endfunction()
 function(setup_cocos_app_config app_name)
```

Comment:
Consider remove this function

Reply:
all configs in `setup_cocos_app_config ` is stable, user don't need to custom it, and moving it up to root CMakeLists.txt will cause much repeat.